soc/v2/ti_k3/am6x/Kconfig

```diff
@@ -2,10 +2,12 @@
 # SPDX-License-Identifier: Apache-2.0
 
 config SOC_SERIES_AM6X_A53
-	bool "TI AM6X A53 Core Series"
 	select SOC_FAMILY_TI_K3
 	select ARM64
 	select CPU_CORTEX_A53
 	select ARM_ARCH_TIMER if SYS_CLOCK_EXISTS
 	help
 	  Enable support for AM6X A53 Series.
+
+config SOC_AM6234_A53
+	select SOC_SERIES_AM6X_A53
```

Comment:
I think this selection should be in a `Kconfig.soc` so it also is true when sourcing in sysbuild.